src/linear_algebra/orientation.lean

```diff
@@ -351,6 +351,14 @@ variables {R} {ι : Type*} [fintype ι] [decidable_eq ι]
 protected def orientation [nontrivial R] (e : basis ι R M) : orientation R M ι :=
 ray_of_ne_zero R _ e.det_ne_zero
 
+lemma orientation_map [nontrivial R] [is_domain R] (e : basis ι R M)
+  (f : M ≃ₗ[R] M) : (e.map f).orientation = (f.det)⁻¹ • e.orientation :=
+begin
+  simp_rw [basis.orientation, smul_ray_of_ne_zero, ray_eq_iff],
+  rw [(e.map f).det.eq_smul_basis_det e, e.det_map, ←linear_equiv.coe_coe, e.det_comp f.symm e,
+      e.det_self, mul_one, units.smul_def, linear_equiv.coe_inv_det],
+end
```

Comment:
I need to wait for a cache to prove it, but I now realize this statement should instead be the more general:
```suggestion
lemma orientation_map [nontrivial R] [is_domain R] (e : basis ι R M)
  (f : M ≃ₗ[R] N) : (e.map f).orientation = e.orientation.map f :=
sorry
```
along with elsewhere:
```lean
lemma map_automorphism (f : M ≃ₗ[R] M) : o.map f = (f.det)⁻¹ • f := sorry
```